addons/test_mail/tests/test_performance.py

```diff
@@ -522,7 +522,7 @@ def test_mail_composer_w_template_form_attachments(self):
     def test_message_assignation_email(self):
         self.user_test.write({'notification_type': 'email'})
         record = self.env['mail.test.track'].create({'name': 'Test'})
-        with self.assertQueryCount(__system__=28, employee=29):  # tm 27/28
+        with self.assertQueryCount(__system__=29, employee=30):  # tm 28/29
```

Comment:
Still valid ? Do you know why we have an additional query ?

Reply:
It is because we have integrated the "internal communication" banner from "message_notification_email" into the new default template "mail_notification_layout" which cause one added query in the test for querying the flag internal from mail.message.subtype.
In the template, we have:
```xml
<t t-set="subtype_internal" t-value="subtype and subtype.internal"/>
...
<p t-if="subtype_internal" style="background-color: #f2dede; padding: 5px; margin-bottom: 16px; font-size: 13px;">
    <strong>Internal communication</strong>: Replying will post an internal note. Followers won't receive any email notification.
</p>
```
Note that here I have just changed 
```xml
<t t-set="subtype_internal" t-value="subtype.internal"/>
```
to 
```xml
<t t-set="subtype_internal" t-value="subtype and subtype.internal"/>
```
As subtype is not a non-null field.

Reply:
That's ok I guess ^^